### PR TITLE
New package: xephem-3.7.7

### PR DESCRIPTION
**Pull request by JohnKallimanis**

### CODE REVIEW

**Comment by ericonr** [comment]
Does this work with new `mostif` package?

And yes, most `-devel` dependencies go in `makedepends`.

https://www.clearskyinstitute.com/xephem/ seems to indicate MIT licensing...

**Comment by JohnKallimanis** [comment]
Have not checked compatibility with mostif. 
The license seems to have changed to MIT. Previous versions had a more restrictive license that allowed modification only for porting purposes. Indeed, the previous license was:
https://github.com/JohnKallimanis/johnnywho/blob/master/licences/xephem

File: srcpkgs/xephem/template [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+pkgname=xephem
+version=3.7.7
+revision=1
+build_style=gnu-makefile
+make_build_args="-C GUI/xephem"
+makedepends="libpng-devel lesstif-devel libICE-devel libSM-devel libX11-devel libXau-devel
+	libxcb-devel libXdmcp-devel libXmu-devel libXp-devel libXt-devel libXext-devel groff"
+short_desc="Interactive astronomical ephemeris for the serious amateur astronomer"
+maintainer="John Kallimanis <johnkall007@gmail.com>"
+license="Proprietary"
```
Comment (Chocimier) [inline srcpkgs/xephem/template:11]: Actually regular MIT license.

**Comment by JohnKallimanis** [comment]
At some point there was a clause forbidding commercial use. It may have
been lifted from some version onwards.

**Pull request closed without merging**